compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/platform/CompositionLocals.kt

```diff
@@ -188,31 +188,43 @@ internal val LocalPointerIconService = staticCompositionLocalOf<PointerIconServi
 @Composable
 internal fun ProvideCommonCompositionLocals(
     owner: Owner,
+    isMainOwner: Boolean,
     uriHandler: UriHandler,
     content: @Composable () -> Unit
 ) {
     CompositionLocalProvider(
-        LocalAccessibilityManager provides owner.accessibilityManager,
-        LocalAutofill provides owner.autofill,
-        LocalAutofillTree provides owner.autofillTree,
-        LocalClipboardManager provides owner.clipboardManager,
-        LocalDensity provides owner.density,
+        // Locals that depend on the owner itself should be overridden always (regardless of whether
+        // this owner is the main one) are ones that .
         LocalFocusManager provides owner.focusOwner,
-        @Suppress("DEPRECATION") LocalFontLoader
-            providesDefault @Suppress("DEPRECATION") owner.fontLoader,
-        LocalFontFamilyResolver providesDefault owner.fontFamilyResolver,
-        LocalHapticFeedback provides owner.hapticFeedBack,
-        LocalInputModeManager provides owner.inputModeManager,
-        LocalLayoutDirection provides owner.layoutDirection,
-        LocalTextInputService provides owner.textInputService,
-        LocalPlatformTextInputPluginRegistry provides owner.platformTextInputPluginRegistry,
-        LocalTextToolbar provides owner.textToolbar,
-        LocalUriHandler provides uriHandler,
-        LocalViewConfiguration provides owner.viewConfiguration,
         LocalWindowInfo provides owner.windowInfo,
         LocalPointerIconService provides owner.pointerIconService,
-        content = content
-    )
+    ){
```

Comment:
Please apply auto-format
```suggestion
    ) {
```